docs/basics/functions.md

```diff
@@ -665,6 +669,109 @@ Which would return the following:
 "metrics": []
 ```
 
+### sample_include_filter
+
+| Applies to | Description |
+| :--------- | :---------- |
+| API | only creating the sample if both a key and value is found in the sample |
+|| in absence of sample_include_filter, it defaults to include all|
+|| if the sample cannot pass sample_include_filter filter, the sample_filter and sample_exclude_filter will not be evalucated|
+
+**Example**
+
+Consider a service that returns the following payload:
+```json
+{
+  "usageInfo": [
+    {
+      "quantities": 10,
+      "customerId": "abc"
+    },
+    {
+      "quantities": 20,
+      "customerId": "xyz"
+    }
+  ]
+}
+```
+
+You could completely skip creating the output sample:
+
+```yaml
+name: example
+apis:
+    - name: someService
+      url: http://some-service.com/samples
+      sample_include_filter:
+        - name: abc
+```
+
+Which would return the following:
+
+```json
```

Comment:
Can you remove the extra spacing here, ie, align to the left?

Reply:
Updated. thanks!